anta/runner.py

```diff
@@ -13,6 +13,9 @@
 
 logger = logging.getLogger(__name__)
 
+# Key from YAML file tranfered to AntaTestTemplate of the test.
+TEST_TPL_PARAM = "tpl_options"
+
```

Comment:
Should we use "template_params" instead of "tpl_options"? Just to be consistent?

I would also put an S ; TEST_TPL_PARAMS

Reply:
Let's discuss about this option name in GHC